cscs-checks/microbenchmarks/stream/stream.py

```diff
@@ -24,12 +28,11 @@ def __init__(self):
             'PrgEnv-cray': ['-homp'],
             'PrgEnv-gnu': ['-fopenmp', '-O3'],
             'PrgEnv-intel': ['-qopenmp', '-O3'],
-            'PrgEnv-pgi': ['-mp', '-O3']
+            'PrgEnv-pgi': ['-mp', '-O3'],
+            '*': ['-O3']
```

Comment:
This has no effect. This is a normal dictionary. You should do handle this case in the setup method:

```python
        self.build_system.cflags = self.prgenv_flags.get(envname, ['-O3'])
```